.pylintrc

```diff
@@ -1,17 +1,18 @@
 [MESSAGES CONTROL]
 # For all codes, run 'pylint --list-msgs' or go to 'http://pylint-messages.wikidot.com/all-codes'
 disable=
-    invalid-name,
+    cyclic-import,
```

Comment:
I recommend adding explanation of why disable them.

Reply:
Good suggestion, but let's keep it for now as this is only for `azdev`. CLI's `pylintrc` has detailed explanation.

Reply:
I have removed unused items. The rest of them is self-explanatory.